services/skipchain/skipchain.go

```diff
@@ -443,30 +443,36 @@ func (s *Service) bftVerify(msg []byte, data []byte) bool {
 				return true
 			}
 		}
+	default:
+		f, ok := verifiers[sb.VerifierID]
```

Comment:
*Discussion*: Maybe it's not possible directly or with too much work I don't know but since we introduce the registration of verification functions, it seems best to 
+ delete the two above `case` 
+ put their logic into separate verification functions 
+ register these verification functions in a `init()` or so
+ get rid of this switch case 

That way all is "unified" according to this new way of using verification functions (and in a few months we won't be like `whut ? Why is it not used in a registration function ... ??` ).
 What do you think ? From just what I see now it does not seem to be too much work but @ineiti you know better this code so what's your thoughts on this ?